src/main/java/SpringAIController.java

```diff
@@ -0,0 +1,45 @@
+import com.openai.services.AIServiceHelper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
```

Comment:
Avoid using wildcard imports. Import only specific classes needed from the package.